app/views/layouts/application.html.erb

```diff
@@ -66,6 +66,12 @@
       <% if flash[:info] %>
         <p><%= flash[:info] %></p>
       <% end %>
+        <div id="flashes">
+          <% flash.each do |name, msg| %>
+          <%= content_tag :div, raw(msg), id: "flash_#{name}",
```

Comment:
Yeah, I think devise passes some html message occasionally that we don't want to escape.